### ISSUE TITLE
The training of PartCrafter

### ISSUE THREAD

**Issue by zhirui-gao** [wgsxm/PartCrafter#8]
Thank you for your amazing work on PartCrafter! I have a question regarding the training process.

In the paper, you mentioned the training details of  ​​rectified flow matching​​, and I'm curious about how the ​​latent codes​​ for each part are obtained. Did you fine-tune the VAE component, or did you directly utilize the VAE from ​​TripoSR​​?

**Comment by chenguolin** [comment]
Hi @zhirui-gao, Thanks for your interest.

Each part is encoded into latent vector set independently. Importantly, parts are **NOT normalized** to the center and unit scale. VAE is not finetuned: it directly utilizes the VAE from TripoSR. Finetuning the VAE will probably get better performance, and we might try it in the future.

**Comment by zhirui-gao** [comment]
Thanks for your timely reply!
